src/components/MapCard.js

```diff
@@ -281,14 +281,15 @@ export default class MapCard extends LitElement {
     });
   }
 
-  setConfig(inputConfig) {
-    this.config = new MapConfig(inputConfig);    
+  setConfig(config) {
+    this.config = config;

```

Comment:
isn't this a bit odd? Might be better to keep L286 + super.setConfig(config)

Reply:
Reading more I'm not sure the this.config = config bit is needed at all, as lit manages the main config as an attribute already.

Reply:
Let's do a call to `super` just to be sure.

Reply:
I've just double checked,  `setConfig` doesn't appear to exist on the underlying Lit component so calling super just throws an error. As I understand it, its a specific requirement from HA that gets called into when booting, so it's super is `undefined` unless you inherit from a HA specific boiler plate that implements its own.

I did also try without setting the `this.config = config;` entirely, but it appears that rebreaks card_mod.